go.mod

```diff
@@ -5,7 +5,7 @@ go 1.13
 require (
 	github.com/ElrondNetwork/big-int-util v0.1.0
 	github.com/ElrondNetwork/elrond-go-logger v1.0.4
-	github.com/ElrondNetwork/elrond-vm-common v0.3.4-0.20210707100510-6cef7cea933f
+	github.com/ElrondNetwork/elrond-vm-common v1.0.0
```

Comment:
you hae to referentiate arwen v1_3 at line 21 with the latest commit hash

Reply:
done.